Description: Don't fail (-Werror) on "unused-variable" and "maybe-uninitialized" warnings.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/1483400

--- a/SConstruct
+++ b/SConstruct
@@ -1734,6 +1734,10 @@
         # than the return type.
         AddToCXXFLAGSIfSupported(myenv, "-Wredundant-move")
 
+        # Debian
+        AddToCCFLAGSIfSupported(myenv, "-Wno-unused-variable")
+        AddToCCFLAGSIfSupported(myenv, "-Wno-maybe-uninitialized")
+
     # Check if we need to disable null-conversion warnings
     if myenv.ToolchainIs('clang'):
         def CheckNullConversion(context):
